shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -923,7 +932,13 @@ - (void)setSelectedTextRangeLocal:(UITextRange*)selectedTextRange {
 
 - (void)setSelectedTextRange:(UITextRange*)selectedTextRange {
   [self setSelectedTextRangeLocal:selectedTextRange];
-  [self updateEditingState];
+
+  if (_enableDeltaModel) {
+    self.currentTextEditingDelta = [[FlutterTextEditingDelta alloc] initWithNonText:self.text];
+    [self updateEditingStateWithDelta];
```

Comment:
It looks like whenever you call `updateEditingStateWithDelta` you actually have access to the delta. Is it possible to make `updateEditingStateWithDelta` take the delta as a parameter then you don't have to use instance variables to track them?

Reply:
I think this is possible. But like this I wouldn't be able to verify the delta in the tests I don't think?